eduNEXT/ednx-saas-themes
# feat: update css-runtime to with paragon 22.0.0-alpha.13

This PR updates the files created with paragon for Bragi default theme. 

How to test 

Add the following configuration to tenant config:
```
    "THEME_OPTIONS": {
        "theme": {
            "name": "css-runtime",
            "parent": "bragi"
        }
    }
```
Test the environment looks same as bragi by default and check the list of variables in the inspect.

Finally add :
`    "css_overrides_file_extra": "https://css-varsify.s3.amazonaws.com/public/ab931927-1d07-4bca-97bb-e86e5d248a6f.css"
`

To the tenant config to verify is reading an override file.


[JIRA](https://edunext.atlassian.net/browse/DS-737)
